backend/src/jwt/decode.spec.js

```diff
@@ -69,24 +69,25 @@ describe('decode', () => {
           {
             role: 'user',
             name: 'Jenny Rostock',
-            avatar: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
             id: 'u3',
             slug: 'jenny-rostock',
           },
           {
+            image: Factory.build('image', {
+              url: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
+            }),
             email: 'user@example.org',
           },
         )
       })
 
-      it('returns user object except email', async () => {
+      it('returns user object without email', async () => {
         await expect(decode(driver, authorizationHeader)).resolves.toMatchObject({
           role: 'user',
           name: 'Jenny Rostock',
-          avatar: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
           id: 'u3',
-          email: null,
           slug: 'jenny-rostock',
+          // email: undefined,
```

Comment:
?

Reply:
The `authorizationHeader` does not contain the email anymore. It's a small code creep when I removed `.avatar` from the list of attributes passed to `encode`.

Reply:
I'm going to remove. I thought it could be a hint that `email` is not included in the token.